### PR TITLE
Feature/timestamps

### PR DESCRIPTION
**Pull request by JWillegers**
Changing timestamp to unix timestamp in centiseconds.
This allows for the packets saved on the SD card to be traced back to real world time.
The unit centiseconds was chosen to be precise enough (being able to run at 100Hz) but also use a minimal amount of bytes.

Also changed the generator such that uint64_t to be decode correctly in .h files

### CODE REVIEW

File: generator/Generator.py [hunk 1]
```diff
@@ -399,8 +399,15 @@ def to_getter(self, packet_name:str, variable:Variable, current_bit:int) -> str:
...
 				shift_to_left = n_bits_remaining - bits_from_byte
-
-				bitwise_operation = f"(({payload_variable}[{at_byte}]{bitshift_mask}){shift(shift_to_left, shift_to_right)})"
+				# for C, figure out if type casting is needed
+				type_cast = ""
+				if type(self) == C_Generator:
+					if shift_to_left >= 32:
+						type_cast = "(uint64_t) "
+					elif shift_to_left >= 24:
+						type_cast = "(uint32_t) "
```
Comment (emielsteerneman) [inline generator/Generator.py]: Should be placed within the `C_Generator` class if you want to be consistent

Comment (emielsteerneman) [inline generator/Generator.py]: This feels weird to me since the system is already 32 bits. Should not be needed

**Review by emielsteerneman — CHANGES_REQUESTED** [review]

File: generator/Generator.py (hunk 1 above)
Comment (JWillegers) [reply generator/Generator.py]: It is needed.

Before the changes the timestamp looks like this, corresponding to Dec 07 2023 14:26:10: 
image 
from [REM_Robotfeedback before changes.csv](https://github.com/RoboTeamTwente/roboteam_embedded_messages/files/13612275/REM_Robotfeedback.before.changes.csv)

After your suggested changes the timestamps looks like this, corresponding to a date in 2317:
image
from [REM_RobotFeedback after change.csv](https://github.com/RoboTeamTwente/roboteam_embedded_messages/files/13612290/REM_RobotFeedback.after.change.csv)

Comment (emielsteerneman) [reply generator/Generator.py]: Still not sure why it is needed exactly but alright if it works, it works.

Comment (JWillegers) [reply generator/Generator.py]: Resolved in commit c187413

**Review by emielsteerneman — APPROVED** [review]
lgtm

**Pull request merged**